components/BookstoreItem.vue

```diff
@@ -65,11 +79,26 @@ const nftStore = useNFTStore()
 const metadataStore = useMetadataStore()
 const bookInfo = useBookInfo({ nftClassId: props.nftClassId })
 const bookCoverSrc = computed(() => getResizedImageURL(bookInfo.coverSrc.value || props.bookCoverSrc, { size: 300 }))
+const { isLikerPlus, plusDiscountRate } = useSubscription()
 
 const bookName = computed(() => bookInfo.name.value || props.bookName)
 const authorName = computed(() => bookInfo.authorName.value)
 
-const formattedPrice = computed(() => formatPrice(props.price))
+const plusPrice = computed(() => {
+  const items = bookInfo.pricingItems.value
+  if (!items.length) return -1 // All items are unlisted
+  const minPrice = Math.min(...items.map(item => item.price))
+  return Math.round(minPrice * (1 - plusDiscountRate))
+})
+
+const shouldApplyPlusDiscount = computed(() => {
+  return !!isLikerPlus.value && plusPrice.value > 0
+})
+
+const formattedPrice = computed(() => {
+  const price = shouldApplyPlusDiscount.value ? plusPrice.value : props.price
+  return formatPrice(price)
+})
```

Comment:
Not sure if splitting `formattedPrice` and `formattedPlus/DiscountPrice` would simplify `<template>` logic above, your call.